game/src/main/scala/io/aigar/game/Grid.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace._
+import java.util.Random
+
+object Grid {
+  final val WidthPerPlayer = 100
+  final val HeightPerPlayer = 100
+}
+class Grid(val width: Int, val height: Int) {
+  val random = new Random()
```

Comment:
While reading java's doc `The class uses a 48-bit seed, which is modified using a linear congruential formula. (See Donald Knuth, The Art of Computer Programming, Volume 2, Section 3.2.1.)`. 

Is there a reason to not seeding it?